contents/blog/what-is-a-growth-engineer.md

```diff
@@ -0,0 +1,108 @@
+---
+title: What is a growth engineer (and how they help your startup)
+date: 2023-04-19
+author:
+    - ian-vanagas
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+ - Explainers
+---
+
+Every startup loves to see signups, subscriptions, and revenue go up, yet all engineers basically work on things indirectly related to doing so. The power of growth engineers is that they focus on making these numbers go up, working on whatever it takes.
+
+A growth engineer is an engineer who works on projects impacting the core business metrics, no matter where they are in the product. This is different than regular engineers who work on specific products and features to improve metrics for them. This indirectly improves the core business metrics.
+
+Growth engineers help companies grow faster by focusing on projects with high potential to impact core business metrics. We know this because we have a team of growth engineers at PostHog, and they’ve made a big impact on signups and revenue. 
```

Comment:
I'm not sure if this belongs here but we have a very specific way of doing growth at PostHog that is different from what you might find at another company. Examples include:

- We don't take on special projects within the company's _products_, even if we think there is potential for this projects to grow revenue, etc.
  - This is because the product teams should be doing the innovation within their product areas, and they should know their customers best.
- We don't really focus on top-of-funnel at the moment - our marketing team is _very_ good at that
- We don't focus on activation for any particular product, but we (plan to!) help the product teams understand how to measure activation and give them the tools to improve it.
  - We are fully responsible for _global_ activation, which is specific to multi-product companies.

see: https://posthog.com/handbook/small-teams/growth

Reply:
Yeah I'm trying to strike a balance between what we do at PostHog, and what growth engineering means more generally. I'll work to make it clearer.